src/compiler/compile/Component.ts

```diff
@@ -1284,10 +1284,23 @@ export default class Component {
 				const module_dependencies = new Set<string>();
 
 				let scope = this.instance_scope;
+				const {declarations: outsetScopeDecalarations} = this.instance_scope;
```

Comment:
`outsetScopeDecalarations` -> to snake case

the naming convention is to use `snake_case`